node/configuration.yaml

```diff
@@ -305,6 +307,8 @@ mainnet_base: &mainnet_base
     blockRetrievalQueueSize: 100
     propagationQueueSize: 100
     pendingTxResubmissionPeriod: 7 # seconds
+    walletProductionApi: false
```

Comment:
This should be `true`, as per https://github.com/input-output-hk/cardano-sl/pull/2213/commits/f515bcb1e65ebc61df9fdf0a5d3020a26066b7b0 which was to be squashed in here.

Reply:
@dcoutts Dang, I think I have mixed up the two (aka I have flipped the one on line `121` to `true`.

Reply:
Erm, maybe I'm the one getting confused. The dev one should be false, and the mainnet one should be true. So that looks right now.

Reply:
You are not! This is GH to blame. The fact I have force-pushed eschewed the lines your comment was attached :)